internal/bgp/frrk8s/frrk8s.go

```diff
@@ -233,6 +235,11 @@ func (sm *sessionManager) updateConfig() error {
 			}
 			portUint16 := uint16(portUint)
 
+			password := s.Password
```

Comment:
still not sure I like this.

Possible alternatives:

- we don't translate secret -> password in frrk8s mode
- we have two different fields, one with the "to be used in local bgp (frr / native)", and the other which is the password to propagate